adobe/aio-cli-plugin-api-mesh
# Updated init to support Quick Start setup

## Description

In an effort to make it easy for mesh developers to get started we have embarked on a quick start template to setup and test meshes both on the local and on github codespaces.

This PR adds the same template files to the init command to make the output of init command to look similar to the output of the template.

Draft template: https://github.com/revanth0212/api-mesh-starter-kit

## Related Issue

https://jira.corp.adobe.com/browse/CEXT-3347

## Motivation and Context

Better DevEx

## How Has This Been Tested?

This structure has already been validated on the quick start template.

Steps:

1. Run init command
2. Update mesh.json/.env with the necessary details
3. Run start command `yarn start mesh.json`
4. Alternatively run `aio api-mesh run mesh.json`
5. Run debug command `yarn debug mesh.json`
6. Alternatively run `aio api-mesh run mesh.json --debug`
7. Upload the init folder to github and create a codespace
8. Run start and debug command to verify the same behavior (aio api-mesh run does not work directly on codespaces)

## Screenshots (if appropriate):

## Types of changes




## Checklist:
